### PR TITLE
fix: don't show withdrawn collections on reports unless report is withdrawn itself

### LINKED ISSUE #3196: Filter out withdrawn collections in the biobank and collection views
If the collection has status withdrawn it needs to be filtered out in the biobank details view and collection details view (for subcollections that are withdrawn).
In the the collection details view there needs to be a banner that says that the collection is withdrawn.

### PR DESCRIPTION
**Pull request by chinook25**
fixes #3196 
how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added tests

### CODE REVIEW

File: apps/directory/vite.config.js [hunk 1]
```diff
@@ -5,10 +5,9 @@ import { createHtmlPlugin } from "vite-plugin-html";
 import monacoEditorPlugin from "vite-plugin-monaco-editor";
 
 // eslint-disable-next-line no-undef
-const HOST =
-  process.env.MOLGENIS_APPS_HOST || "https://bbmri-emx2-test.molgeniscloud.org";
+const HOST = process.env.MOLGENIS_APPS_HOST || "http://localhost:8080";
 // eslint-disable-next-line no-undef
-const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "Directory";
+const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "aDirectory";
```
Comment (esthervanenckevort) [inline apps/directory/vite.config.js:10]: What is this change? Should this be in this PR

File: apps/directory/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "include": ["./src/**/*"],
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "strict": true,
+    "isolatedModules": true,
+    "allowJs": true,
+    "types": ["vite/client"]
+  }
+}
```
Comment (esthervanenckevort) [inline apps/directory/tsconfig.json:10]: I guess this is because you converted the code to TypeScript?

**Review by esthervanenckevort — CHANGES_REQUESTED** [review]
When I try to open the biobank view it fails and I get a blank screen

**Review by mswertz — COMMENTED** [review]
happy with this PR; need Esther to approve the functionality.

File: apps/directory/vite.config.js (hunk 1 above)
Comment (chinook25) [reply apps/directory/vite.config.js:10]: Accidentally committed this, will revert

**Review by dtroelofsprins — APPROVED** [review]
- Withdrawn (sub-)collections are not shown in on the biobank overview on the main page, and also not on the biobank card 
- Withdrawn sub-collections are not shown on the collection card.

Only thing is that if a collection is withdrawn, and the sub-collection not, it's shown on the biobank overview on the main page, but not in the biobank card. Decided that this is an edge case that actually should not occur, as if a collection is withdrawn, all of its sub-collections should also be withdrawn. So that's then a data issue.

**Pull request merged**